parlai/scripts/build_candidates.py

```diff
@@ -43,7 +43,7 @@ def build_cands(opt):
 
     print('[ starting to build candidates from task.. (ex:' + str(num_examples) + ')]')
     print('[ saving output to {} ]'.format(outfile))
-    cands = []
+    cands = set()
```

Comment:
This is fine, but why was it made a set?

Reply:
so we don't have repeat utterances in fixed set which is both slower for no reason, and also produces some problems when the topk is all the same